virttest/qemu_monitor.py

```diff
@@ -316,8 +316,8 @@ def _log_command(self, cmd, debug=True, extra_str=""):
         :param extra_str: Extra string would be printed in log.
         """
         if self.debug_log or debug:
-            logging.debug("(monitor %s) Sending command '%s' %s",
```

Comment:
As the bug described, we had better to add log as belows.   
e.g.
    (virt-tests-vm1.qmpmonitor1)
    (virt-tests-vm1[dst-xxx].qmpmonitor1)
 
So  I think it is better to changed as below.
+            logging.debug("(monitor %s.%s) Sending command '%s' %s",
+                          self.vm.name, self.name, cmd, extra_str)